### PR TITLE
Renaming part 1

### PR DESCRIPTION
**Pull request by Shillaker**

### CODE REVIEW

File: bin/build_tflite_native.sh [hunk 1]
```diff
@@ -15,7 +15,9 @@ TF_LITE_DIR=${TF_DIR}/tensorflow/lite
 
 pushd ${TF_LITE_DIR}
 
-make -j \
+N_PROC=$(nproc --ignore=1)
+
+make -j ${N_PROC} \
```
Comment (mfournial) [inline bin/build_tflite_native.sh:20]: :+1:

Comment (Shillaker) [reply bin/build_tflite_native.sh:20]: Added after almost bricking my machine :smile:

Comment (mfournial) [reply bin/build_tflite_native.sh:20]: There is too many fork bombs in this project!
```
git grep -- "-j[ \"]?[^\d]" 
ansible/assemblyscript.yml:      shell: "npm install -g uglify-js@1"
ansible/roles/openmpi/tasks/install.yml:    - "make -j"
ansible/roles/pistache/tasks/main.yml:    - "make -j"
ansible/roles/protobuf/tasks/main.yml:    - "make -j"
ansible/roles/rapidjson/tasks/main.yml:  shell: "make -j"
ansible/roles/spdlog/tasks/main.yml:    - "make -j"
bin/build_genomics_native.sh:make -j
bin/build_polybench_native.sh:cmake --build . --target polybench_all_funcs  -- -j
bin/build_prk_native.sh:make allmpi -j
bin/build_tflite_native.sh:make -j \
bin/netns_inner.sh:    iptables -t nat -I POSTROUTING -s ${base_ip}/24 -o eth0 -j MASQUERADE
bin/netns_inner.sh:    iptables -I FORWARD -i eth0 -o ${vif_name} -j ACCEPT
bin/netns_inner.sh:    iptables -I FORWARD -o eth0 -i ${vif_name} -j ACCEPT
bin/python_profile.sh:perf inject -i perf.data -j -o perf.data.jitted
docs/debugging.md:perf inject -i perf.data -j -o perf.data.jitted
faasmcli/faasmcli/tasks/compile.py:    cmd = "make {}".format(target) if target else "make -j"
faasmcli/faasmcli/tasks/knative.py:        make_cmd = "cmake --build . --target {} -- -j".format(target)
faasmcli/faasmcli/tasks/libs.py:    res = call("make -j", shell=True, cwd=build_dir)
faasmcli/faasmcli/tasks/libs.py:    make_cmd = "make -j {}".format(make_target)
```

Comment (Shillaker) [reply bin/build_tflite_native.sh:20]: Oh man, that's sad :frowning_face: will fix as I go

**Pull request merged**